### ISSUE TITLE
[colorize_nicks.pl] Doesn't colorize the nicks in the latest versions of irssi-git

### ISSUE THREAD

**Issue by g4570n** [irssi/scripts.irssi.org#708]
```
Irssi-git: 1.3-dev-277-g784373a7-obs
colorize_nicks.pl: 0.3.7
nickcolor_expando.pl: 0.4.0
```
I detected that in the latest versions of irssi-git, the color of the nicks has stopped showing in the body of chat messages.

screenshot: https://0x0.st/i_q5.png

**Comment by g4570n** [comment]
I'm sorry, I didn't see that an issue had already been created in the `irssi` repo to solve this problem

https://github.com/irssi/irssi/issues/1189

**Comment by g4570n** [comment]
Solved in `irssi-git: 1.3-dev-281-g5ba58d90-obs` and `colorize_nicks.pl: 0.4.0`

Thanks! :)
